### ISSUE TITLE
Release on pypi?

### ISSUE THREAD

**Issue by haixuanTao** [huggingface/parler-tts#123]
Just wanted to say that the speech-to-speech demo is super impressive! 

I think it could be a good time to release a version on pypi so that people can easily install parler-tts as well as be bulletproof to breaking change in the main repo.

For example I plan on integrating the latest kvcache from @eustlb  within dora, but would want to make sure that people are using the right version of parler-tts. See: https://github.com/huggingface/parler-tts/pull/89

I can help with the github action if needed, but might not be the most suited person.

**Comment by ylacombe** [comment]
Hey @haixuanTao, thanks for your feedback !

I think you might be right ! However, I'm not sure yet how to proceed to do a proper release, any pointers?

**Comment by haixuanTao** [comment]
Yes, so the things you want to do is set up a GitHub action. And make the release as automatic as possible. No manual action, except a click. 

There's couple of example and I'll reckon testing on pre release.

You want to make sure to respect version semantic so to make sure that api breaking change are well supported. 

I would also recommend making sure you can export using Python ABI so that you don't have to worry about Python version.

Last but not least, I would recommend checking import os calls as to make sure it's cross platform, cross architecture. 

That's it
